.gitignore

```diff
@@ -93,3 +93,6 @@ staticfiles/
 release-notes-checklist
 
 static/scss/material.css
+
+# Local settings
+micromasters.yml
```

Comment:
This file is an alternate means of specifying environment variables.  This comes from the django cookiecutter and is defined in settings.py